### PR TITLE
Update/prop types

### PR DESCRIPTION
**Pull request by boygirl**
cc/ @sto3psl  I think this PR fixes the last of the the compatibility issues for React 15.5

### CODE REVIEW

File: src/victory-portal/portal.js [hunk 1]
```diff
@@ -1,4 +1,6 @@
-import React, { PropTypes } from "react";
+import React from "react";
+
```
Comment (sto3psl) [inline src/victory-portal/portal.js:2]: This empty line is not really necessary.

File: demo/app.js [hunk 2]
```diff
@@ -1,42 +1,63 @@
 /*global document:false */
+/*global window:false */
```
Comment (ryan-roemer) [inline demo/app.js]: NIT: Suggest inline, comma separate with previous.

File: test/client/spec/victory-util/prop-types.spec.js [hunk 3]
```diff
@@ -23,29 +32,30 @@ describe("prop-types", () => {
 
     beforeEach(() => {
       sandbox = sinon.sandbox.create();
-      sandbox.stub(console, "error");
+      sandbox.spy(console, "warn");
```
Comment (ryan-roemer) [inline test/client/spec/victory-util/prop-types.spec.js]: GLOBAL -- We want `stub`s right? We don't want output all over our tests..

**Review by ryan-roemer — COMMENTED** [review]
Side note -- this is going to collide with my deps + eslint work that's refactoring various stuff, but I'll take the hit and clean up / merge resolve after you...

**Review by sto3psl — CHANGES_REQUESTED** [review]
This looks fine to me and can be merged if Travis passes. We can also put 

closes #232

into the description to automatically close my PR.

**Comment by boygirl** [comment]
@ryan-roemer should the `prop-types` package previously supplied by `react` be in `peerDependencies` or regular `dependencies`?

**Comment by boygirl** [comment]
@sto3psl actually, I think this is going to be a major version change, as it will be a breaking change for users who aren't including `prop-types` in their projects. I'm going to delay putting this in for that reason.

**Comment by sto3psl** [comment]
@boygirl Is it really? A lot of other libs (like [react-virtualized](https://github.com/bvaughn/react-virtualized/blob/master/CHANGELOG.md#971)) updated to `prop-types` with a patch release. It's also not really hard to "fix" it and everyone should transition to the new package. In production it's also not a problem since you should strip `PropTypes` there anyway.

**Pull request merged**